acceptable-albatrosses/albatrosses_hub/forums/tests.py

```diff
@@ -0,0 +1,124 @@
+from datetime import datetime
+from django.urls import reverse
+from django.test import TestCase, Client, RequestFactory
+from .models import Board, Post
+from ..authentication.models import User
+from .views import PostCreateView, CommentCreateView, BoardCreateView
+
+
+class ForumTestCase(TestCase):
+    """Class to perform unit tests of forum app."""
+
+    def setUp(self):
```

Comment:
You've used it for the Auth tests as well, but it's nice to see these `setUp` methods.